janusgraph-cql/src/main/java/org/janusgraph/diskstorage/cql/CQLStoreManager.java

```diff
@@ -159,7 +158,6 @@
 
     final ExecutorService executorService;
 
-    @Resource
```

Comment:
Is it related to Java 11 upgrade or is it just a cleanup?

Reply:
I don't understand why it is need and starting with java 9 it is moved into an external package

Reply:
I see there is `CQLStoreTest` which injects `CqlSession` into `CQLStoreManager`.

```
    @Mock
    private CqlSession session;

    @InjectMocks
    private CQLStoreManager mockManager = new CQLStoreManager(getBaseStorageConfiguration());

```

I assume, it injects using `@Resource` annotation. That said, it looks like all tests passes. So, maybe it can also inject mock using data types comparison of fields of a class. That said, I didn't verify that.
I guess it is safe to be removed but we just need to double check that.

Reply:
Okay, quickly looking at Mockito docs, I confirm that the field will be injected by type.

From [Mockito docs](https://javadoc.io/doc/org.mockito/mockito-core/3.8.0/org/mockito/InjectMocks.html): 
```
Field injection; mocks will first be resolved by type (if a single type match injection will happen regardless of the name), then, if there is several property of the same type, by the match of the field name and the mock name.

Note 1: If you have fields with the same type (or same erasure), it's better to name all @Mock annotated fields with the matching fields, otherwise Mockito might get confused and injection won't happen.

Note 2: If @InjectMocks instance wasn't initialized before and have a no-arg constructor, then it will be initialized with this constructor.
```

Reply:
I see that the usage of `@Resource` annotation is used only here for the whole project.
I could assume that it could be used by IoC somewhere but we don't use any DI frameworks. Also, it's clearly that we violate DI principle inside the constructor as we already creating a new instance of `session` inside the constructor. Thus, I assume `@Resource` here was used to rewrite `session` inside already created `CQLStoreManager`. Probably by some test framework which was using that.
I think it is safe to be removed. At least I didn't found any usage of `@Resource` here.

Reply:
Thank you looking into it.